### ISSUE TITLE
The connection to the server IP:6443 was refused - did you specify the right host or port?

### ISSUE THREAD

**Issue by dmuiX** [weibeld/terraform-aws-kubeadm#16]
```
kubectl --kubeconfig sunny-mongoose.conf get nodes
The connection to the server IP:6443 was refused - did you specify the right host or port?
```

**Comment by weibeld** [comment]
I'm currently investigating the issue.

**Comment by weibeld** [comment]
The issue should now be fixed in [version 0.2.5](https://registry.terraform.io/modules/weibeld/kubeadm/aws/0.2.5) of the Terraform module.

The change consists in switching from Ubuntu 18.04 to Ubuntu 22.10 for the cluster nodes, which solves an [incompatibility issue between containerd 1.5 and Kubernetes v1.26](https://kubernetes.io/blog/2022/11/18/upcoming-changes-in-kubernetes-1-26/#cri-api-removal).
